vendor/engines/secure_rooms/spec/features/manage_card_reader_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "Managing CardReaders" do
+  let(:facility) { create(:setup_facility) }
+  let(:secure_room) { create(:secure_room, facility: facility) }
+  let(:admin) { create(:user, :administrator) }
+  before { login_as admin }
+
+  it "can create, edit, and destroy a card reader" do
```

Comment:
This is probably fine, but I think I'd rather see one spec for each of the three, one for new/create, one for edit/update, and one for destroy.